doc/src/manual/arrays.md

```diff
@@ -519,35 +519,35 @@ Base.IndexStyle{T<:MyArray}(::Type{T}) = IndexLinear()
 This setting will cause `eachindex` iteration over a `MyArray` to use integers. If you don't
 specify this trait, the default value `IndexCartesian()` is used.
 
-### Vectorized Operators and Functions
+### Array and Vectorized Operators and Functions
 
-The following operators are supported for arrays. Also, *every* binary
-operator supports a [dot version](@ref man-dot-operators) that can be
-applied to arrays (and combinations of arrays and scalars) as a
-[fused broadcasting operation](@ref man-vectorized). (For comparison
-operations like `<`, *only* the `.<` version is applicable to arrays.)
+The following operators are supported for arrays:
 
-1. Unary arithmetic -- `-`, `+`, `!`
-2. Binary arithmetic -- `+`, `-`, `*`, `/`, `\`, `^`, `.^`, `div`, `mod`
+1. Unary arithmetic -- `-`, `+`
+2. Binary arithmetic -- `-`, `+`, `*`, `/`, `\`, `^`
 3. Comparison -- `==`, `!=`, `≈` ([`isapprox`](@ref)), `≉`
-4. Unary Boolean or bitwise -- `~`
-5. Binary Boolean or bitwise -- `&`, `|`, `⊻` ([`xor`](@ref))
 
-Some operators without dots operate elementwise anyway when one argument is a scalar:
-`*`, `+`, `-`, and the bitwise operators. The operators `/` and `\` operate elementwise when
-the denominator is a scalar.
+Most of the binary arithmetic operators listed above also operate elementwise
+when one argument is scalar: `-`, `+`, and `*` when either argument is scalar,
+and `/` and `\` when the denominator is scalar, e.g. `[1, 2] + 3`.
```

Comment:
Good call. Perhaps "... when the denominator is scalar. (For example, `[1, 2] + 3 == [4, 5]`.)"? Thoughts? Thanks!